examples/upf_config.json

```
@@ -0,0 +1,15 @@
+[
+    {
+      "hostname": "upf",
```

This would make this example file usable with our default deployment
```suggestion
      "hostname": "upf-external",
```